examples/webpack_cli/index.js

```diff
@@ -0,0 +1,36 @@
+const {
```

Comment:
where did the example come from? maybe attribution in the README

Reply:
👍